### ISSUE TITLE
HTML style nodes with a Font face specified produce an UnicodeDecodeError

### ISSUE THREAD

**Issue by GregSal** [pygraphviz/pygraphviz#365]
The following code produces an error:
```
Gh = pgv.AGraph()
dot_data = '''
digraph "Simple Plan" {
	graph [rankdir=RL];
	node [label="\\N"];
	vv_PlanSetup	[label=<<TABLE>
		<TR><TD><FONT FACE="Times New Roman">Table 1</FONT></TD></TR>
		<TR><TD><FONT FACE="Arial">Column 1</FONT></TD></TR>
		</TABLE>>, shape=plaintext];
	Course	[label=<<TABLE>
		<TR><TD><FONT FACE="Times New Roman">Table 2</FONT></TD></TR>
		<TR><TD><FONT FACE="Calibri">Column 1</FONT></TD></TR>
		</TABLE>>, shape=plaintext];
	vv_PlanSetup -> Course;
}
'''
Gh.from_string(dot_data)
Gh.layout(prog="dot")
Gh.draw("simple_plan.png")
```
while this code works:
```
Gh = pgv.AGraph()
dot_data = '''
digraph "Simple Plan" {
	graph [rankdir=RL];
	node [label="\\N"];
	vv_PlanSetup	[label=<<TABLE>
		<TR><TD><FONT COLOR="red" >Table 1</FONT></TD></TR>
		<TR><TD><FONT COLOR="black">Column 1</FONT></TD></TR>
		</TABLE>>, shape=plaintext];
	Course	[label=<<TABLE>
		<TR><TD><FONT COLOR="red">Table 2</FONT></TD></TR>
		<TR><TD><FONT COLOR="black">Column 1</FONT></TD></TR>
		</TABLE>>, shape=plaintext];
	vv_PlanSetup -> Course;
}
'''
Gh.from_string(dot_data)
Gh.layout(prog="dot")
Gh.draw("simple_plan.png")
```
and produces the following graph:
![simple_plan](https://user-images.githubusercontent.com/33795111/137353347-958ffe95-6d10-4895-9236-559e7e32e134.png)

**Comment by prius** [comment]
Both of your code listings work with python 3.8 and pygraphviz 1.7.

Can you share your environment and a full traceback?

**Comment by GregSal** [comment]
@prius 
Your comment sent me back to look at my code more closely.  In the end I found out that the issue was an invalid UTF-8 character in the FONT FACE section of my code.  I was using Jupyter notebook and it probably crept in while switching platform.  I discovered since posting the issue that what Microsoft calls UTF-8 is actually the UTF-8-BOM (non)standard and it likely stuck a BOM signature onto that line.  Unfortunately I used copy and paste in my testing and ended up copying and pasting the hidden character with it.  When I recreated the code clean from scratch it worked fine just as you said.

In conclusion, this was not a bug in pygraphviz after all.  Sorry for wasting your time.

Greg
